traffic_ops/traffic_ops_golang/topology/validation.go

```diff
@@ -57,9 +57,14 @@ func checkForEdgeParents(nodes []tc.TopologyNode, cachegroups []tc.CacheGroupNul
 	node := nodes[nodeIndex]
 	errs := make([]error, len(node.Parents))
 	for parentIndex := range node.Parents {
-		cacheGroupType := cachegroups[node.Parents[parentIndex]].Type
+		cachegroupIndex := node.Parents[parentIndex]
```

Comment:
This could be a part of the `range` statement above, wherein you'd get the index and the value in the step.

Reply:
Changed in 38838fa76